vault/overlays/nerc-ocp-infra/auto-unseal/cronjob.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: auto-unseal-vault
+spec:
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  schedule: "0 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          name: auto-unseal-vault
+        spec:
+          serviceAccountName: auto-unseal-job
+          restartPolicy: Never
+          containers:
+            - name: auto-unseal-vault
+              image: quay.io/dystewar/vault-unseal:latest
```

Comment:
Before merging this change, we need to:

- Move the sources for this image into the ocp-on-nerc repository
- Configure the image to auto-build on quay (or add workflows to build it on github)
- Update this manifest to use the new image location

Reply:
@larsks understood, any preference where the sources for the custom image live? If not I will place them in a separate dir under auto-unseal. Also I'll set up a workflow to build the image as well

Reply:
what's in the custom image?

Is the correct dckerfile to look at?: https://github.com/dystewart/k8s-vault-auto-unseal/blob/main/docker/Dockerfile

Reply:
Hosting the sources for the custom image in this repository seems ideal.